app/src/views/GrantRound.vue

```diff
@@ -174,7 +186,7 @@ import {
 import { GrantRound, GrantRoundMetadata, Breadcrumb } from '@dgrants/types';
 
 // --- Icons ---
-import { ArrowToprightIcon as ShareIcon } from '@fusion-icons/vue/interface';
+import { TwitterIcon as TweetIcon } from '@fusion-icons/vue/interface';
```

Comment:
^ do we need the rename ? why not use TwitterIcon as is

Reply:
im ok with keep it TwitterIcon. question of taste.